py/study_job_client.py

```diff
@@ -1,160 +0,0 @@
-# Copyright 2019 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
```

Comment:
this file is used in https://github.com/kubeflow/pipelines/blob/master/components/kubeflow/katib-launcher/Dockerfile

Reply:
OK, then we need to keep it. I will add a comment about it.